themes/_core/js/product.js

```diff
@@ -124,6 +124,11 @@ function updateProduct(event, eventType, updateUrl) {
   }
 
   currentRequestDelayedId = setTimeout(function updateProductRequest() {
+    
+    if ((formSerialized == '') || (preview == '')) {
```

Comment:
`if (formSerialized === '' || preview === '') {`

Reply:
Fixed and commit ammended ✅